site/src/Librecores/ProjectRepoBundle/Controller/ProjectController.php

```diff
@@ -316,6 +316,28 @@ public function updateAction(Request $req, $parentName, $projectName)
             [ 'Content-Type' => 'text/plain' ]);
     }
 
+    /**
+     * Delete the Project from the repository
+     *
+     * @param Request $req
+     * @param string $parentName
+     * @param string $projectName
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function deleteAction(Request $req, $parentName, $projectName)
+    {
+        $p = $this->getProject($parentName, $projectName);
+
+        $entityManager = $this->getDoctrine()->getManager();
+        $entityManager->remove($p);
+        $entityManager->flush();
+
+        return $this->render(
+            'LibrecoresProjectRepoBundle:Default:delete.html.twig',[
+                'project' => $p]);
```

Comment:
you just removed this project, you cannot safely pass it to the view as `$p`. Maybe save the properties you need (i.e. project name) to a string before deleting it, and pass that to the view?

Reply:
Fixed this.